viewer/vueapp/src/components/sessions/Sessions.vue

```diff
@@ -1451,6 +1454,16 @@ export default {
         this.loading = false;
       });
     },
+    getCaptureStats: function () {
+      this.$http.get('stats.json')
+        .then((response) => {
+          // TODO ECR - which capture process?
+          this.capStartTime = response.data.data[0].startTime * 1000;
```

Comment:
@awick So, this endpoint returns an array of capture stats. I'm assuming that we should use the first one. Is that a valid assumption?

Reply:
no, you would want to show a little line for each capture instance since they could all be different

Reply:
or maybe only show the most recent or least recent? hmm

Reply:
Ah ok, so there could be multiple restarts. One for each capture at least. So it should probably say in the tooltip: "X capture restarted at this time"